moabb/datasets/physionet_mi.py

```diff
@@ -118,14 +115,16 @@ def _get_single_subject_data(self, subject):
         get_dataset_path(sign, None)
 
         # hand runs
+        idx = 0
         for run in self.hand_runs:
             raw = self._load_one_run(subject, run)
             stim = raw.annotations.description.astype(np.dtype("<U10"))
             stim[stim == "T0"] = "rest"
             stim[stim == "T1"] = "left_hand"
             stim[stim == "T2"] = "right_hand"
             raw.annotations.description = stim
-            data["run_%d" % run] = raw
+            data["run_%d" % idx] = raw
```

Comment:
```suggestion
            data["run_{idx}"] = raw
```
You could use f-strings in MOABB